### PR TITLE
Bodycams and handheld televisions

### PR DESCRIPTION
**Pull request by Aidenkrz**
## About the PR
Added two bodycams, the Security Bodycam, that broadcasts to the security camera subnet, and the NoPro Camera, which broadcasts to the entertainment subnet. Both are loadout options for their respective jobs, (Sec/Reporter)

Also added handheld televisions, which are functionally admin only for now, someone should probably make a construction graph for them and add them to a civ tech for science to research.

AI can NOT view these, as the component doesnt work on clothing/items for some reason
## Media
image

**Changelog**
:cl:
- add: Body cameras are now available for security and reporters in the loadout.
- add: Handheld television are here, you aren't allowed to have them though :)

## Release Notes

- **New Features**
	- Introduced new loadout options including "Bodycam" and "NoPro" featuring surveillance cameras.
	- Added "Reporter outer clothing" loadout group for enhanced customization.
	- New handheld television entity added, functioning as a surveillance camera monitor.

- **Bug Fixes**
	- Updated naming conventions for the handheld power console for consistency.

- **Documentation**
	- Added metadata for texture resources to improve asset management.

- **Chores**
	- Enhanced loadout groups with new items for security and reporter roles.

### CODE REVIEW

**Comment by Verbalase** [comment]
can you actually stop with that dumbass pr

File: Resources/Prototypes/Goobstation/Entities/Objects/Devices/handheld_television.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - type: WirelessNetworkConnection
+      range: 200
+    - type: UserInterface
+      interfaces:
+          enum.SurveillanceCameraMonitorUiKey.Key:
+            type: SurveillanceCameraMonitorBoundUserInterface
+    - type: Speech
+      speechVerb: Robotic
+    - type: SurveillanceCameraSpeaker
+    - type: SurveillanceCameraMonitor
```
[the remark this answers is not in the document]
Comment (Aidenkrz) [reply Resources/Prototypes/Goobstation/Entities/Objects/Devices/handheld_television.yml:30]: you are fucking stupid, you should really fucking shut the hell up unless you know what you're talking about, which you clearly don't. You are literally made of code yet you cant fucking understand the simplest code. Shut yourself down permanently.

**Comment by Aidenkrz** [comment]
> can you actually stop with that dumbass pr

how the fuck is this a dumbass PR

**Comment by Verbalase** [comment]
> > can you actually stop with that dumbass pr
> 
> how the fuck is this a dumbass PR

Not THIS pr. Meant that dumb GMO thing, but I'm going insane...

File: Resources/Textures/Goobstation/Objects/Devices/tablets.rsi/meta.json [hunk 2]
```diff
@@ -1 +1 @@
-{"version":1,"license":"CC-BY-SA-3.0","copyright":"https://github.com/Baystation12/Baystation12/tree/17e84546562b97d775cb26790a08e6d87e7f8077","size":{"x":32,"y":32},"states":[{"name":"tablet"},{"name":"tabletsol"}, {"name":"genericpowerconsole","delays":[[0.3,0.3]]}]}
+{"version":1,"license":"CC-BY-SA-3.0","copyright":"https://github.com/Baystation12/Baystation12/tree/17e84546562b97d775cb26790a08e6d87e7f8077","size":{"x":32,"y":32},"states":[{"name":"tablet"},{"name":"tabletsol"}, {"name":"powerconsole","delays":[[0.3,0.3]]}, {"name":"television","delays":[[0.3,0.3]]}]}
```
Comment (whateverusername0) [inline Resources/Textures/Goobstation/Objects/Devices/tablets.rsi/meta.json:1]: what is this meta :finnadie:

**Review by whateverusername0 — APPROVED** [review]

File: Resources/Textures/Goobstation/Objects/Devices/tablets.rsi/meta.json (hunk 2 above)
Comment (Aidenkrz) [reply Resources/Textures/Goobstation/Objects/Devices/tablets.rsi/meta.json:1]: Ask the person who originally made it

**Review by Piras314 — APPROVED** [review]

**Comment by Aidenkrz** [comment]
@coderabbitai write a walkthrough on this pr

**Pull request merged**